ext/libxml/libxml.c

```diff
@@ -771,11 +771,17 @@ static xmlParserInputPtr php_libxml_external_entity_loader(const char *URL,
 		} else if (Z_TYPE(retval) == IS_RESOURCE) {
 			php_stream *stream;
 			php_stream_from_zval_no_verify(stream, &retval);
-			if (stream == NULL) {
+			if (UNEXPECTED(stream == NULL)) {
```

Comment:
This code fragment just looks like a completely different fix (for stable branches as well)?